modules/test/playwright/tests/headless-builder-web/apiExplorer.spec.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {headlessDiscoveryPagesTest} from '../../fixtures/headlessDiscoveryWebPagesTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {waitForHeadlessBuilderReady} from './utils/headlessBuilder';
+
+export const test = mergeTests(
```

Comment:
The test fails with a clean DB with no FFs enabled. You need to enable the API Builder feature flag here.